python/xorbits/_mars/dataframe/window/rolling/core.py

```diff
@@ -184,6 +184,11 @@ def skew(self, **kwargs):
     def kurt(self, **kwargs):
         return self.aggregate("kurt", **kwargs)
 
+    def corr(self, **kwargs):
+        # for convenience, since the calculation of corr is axis irrelevant.
+        self._axis = 0
```

Comment:
Better to raise an error if use axis is not 0.

Reply:
`axis=1` is also valid.

But when the aggregation function is `corr`, `axis 0` and `axis 1` produce same results.